lib/container.js

```diff
@@ -40,10 +40,16 @@ export class SkygearChatContainer {
    * @param {string} title - string for describing the conversation topic
    * @param {object} meta - attributes for application specific purpose
    * @param {object} [options] - options for the conversation, avaliable options `distinctByParticipants` and `admins`
+   * @param {boolean} [options.distinctByParticipants] - create converastion distinct by participants
+   * @param {[]string|[]User} [options.admins] - admin IDs of the conversation
    *
    * @return {Promise<Conversation>} - Promise of the new Conversation Record
    */
   createConversation(participants, title = null, meta = {}, options = {}) {
+    if (_.isArray(options.admins)) {
```

Comment:
if `options.admins` is not an array, the value will be silently ignored, which seems not desired. 

The `admins` key should also be removed if we copied the keys to the `adminIDs` key.

Reply:
> if options.admins is not an array, the value will be silently ignored, which seems not desired.

But this problem applies to all params 😢